.gitignore

```diff
@@ -7,3 +7,5 @@
 .pytest_cache/
 **__pycache__/
 **db.sqlite3
+.db3
+.sqlite
```

Comment:
The `.sqlite` file extension is included, which is fine if you are using such database files. Ensure that this is intentional and that you actually use `.sqlite` files in your project.